tests/queries/0_stateless/02154_parser_backtracking.sh

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+# Tags: no-tsan
+# ^ TSan uses more stack
```

Comment:
This is likely related to the problem that ConstantNode uses `Field` instead of `IColumn`.